### ISSUE TITLE
[lit | typescript]: CSS variables are not available for ui5 components used inside lit elements

### ISSUE THREAD

**Issue by lp247-SAP** [SAP/ui5-webcomponents#7492]
### Bug Description

Both buttons in the isolated example are rendered differently, although they should look equal. The latter button misses a lot of styles in general. It seems CSS variables are not available for ui5 components inside lit components.

This bug happens for version 1.14 onwards, 1.11 through 1.13 works fine.

### Affected Component

At least button, most likely every component

### Expected Behaviour

Both buttons in the isolated example should render equal.

### Isolated Example

https://codesandbox.io/p/sandbox/funny-scott-djsr7y

### Steps to Reproduce

1. Look into the isolated example.

### Log Output, Stack Trace or Screenshots

N/A

### Priority

High

### UI5 Web Components Version

1.16.0

### Browser

Chrome

### Operating System

MacOS

### Organization

SAP

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by ilhan007** [comment]
Hello @lp247-SAP Peter, 
we did rework our CSS variables loading and placing lately

First we introduced a lot of changes in 1.14 and you started observing the described behaviour.
Recently we merged new significant refactoring with this change https://github.com/SAP/ui5-webcomponents/pull/7449
that we believe should resolve your issue.

It's already available for testing, there is a release candidate version that includes the change
[v1.17.0-rc.1](https://github.com/SAP/ui5-webcomponents/releases/tag/v1.17.0-rc.1)

Is it possible for you to check how it's going with 1.17.0-rc.1?
